tests/drivers/i2s/i2s_api/boards/mimxrt1060_evk.overlay

```diff
@@ -0,0 +1,9 @@
+/*
+* Copyright (c) 2021, NXP
+*
+* SPDX-License-Identifier: Apache-2.0
+*/
+&sai1 {
+    nxp,tx-sync-mode = <0>;
+    nxp,rx-sync-mode = <1>;
```

Comment:
Why is this not set in the board dtsi file

Reply:
this is user configurable, not board depend